string prev_answer
string ques1
string ques2
string ques3
